frontend/public/components/project-overview.jsx

```diff
@@ -11,10 +11,12 @@ import { ResourceLink, resourcePath } from './utils';
 export const ComponentLabel = ({text}) => <div className="co-component-label">{text}</div>;
 
 const ProjectOverviewListItem = ({obj, onClick, selectedItem}) => {
-  const {currentController, kind, metadata} = obj;
+  const {controller, kind, metadata} = obj;
   const {namespace, name, uid} = metadata;
   const isSelected = obj.metadata.uid === _.get(selectedItem, 'metadata.uid', '');
   const className = classnames('project-overview__item', {'project-overview__item--selected': isSelected});
+  const readyPods = obj.status.replicas || obj.status.currentNumberScheduled || 0;
+  const desiredPods = obj.spec.replicas || obj.status.desiredNumberScheduled || 0;
```

Comment:
I'd suggest checking the kind to figure out how to get the replicas.

We are going to have another special case potentially if we add pods with no owner to the overview.

Reply:
OK as a follow on